blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -65,6 +65,20 @@ def sql(url):
             drop(t)
 
 
+@pytest.yield_fixture
```

Comment:
`big_sql` probably has enough data for this test. Also, the dshape and the data do not match up here.